docs/source/en/chat_templating.md

```diff
@@ -758,21 +765,27 @@ it's time to put an end to them!
 If you're unfamiliar with Jinja, we generally find that the easiest way to write a chat template is to first
 write a short Python script that formats messages the way you want, and then convert that script into a template.
 
-Remember that the template handler will receive the conversation history as a variable called `messages`. Each
-message is a dictionary with two keys, `role` and `content`. You will be able to access `messages` in your template
-just like you can in Python, which means you can loop over it with `{% for message in messages %}` or access
-individual messages with, for example, `{{ messages[0] }}`.
+Remember that the template handler will receive the conversation history as a variable called `messages`.  
+You will be able to access `messages` in your template just like you can in Python, which means you can loop over 
+it with `{% for message in messages %}` or access  individual messages with, for example, `{{ messages[0] }}`.
 
 You can also use the following tips to convert your code to Jinja:
 
+### Trimming whitespace
+
+By default, Jinja will print any whitespace that comes before or after a block. This can be a problem for chat
+templates, which generally want to be very precise with whitespace! To avoid this, we strongly recommend writing
+your templates with `{{-` and `{%-` blocks instead of `{{` and `{%`. This will strip any whitespace that comes
```

Comment:
Mentioned above in the Tip already, but not opposed to having it mentioned again!

Reply:
Yep, I know it went in twice - at least that one was deliberate!